openedx/core/djangoapps/api_admin/migrations/0007_auto_20160517_1120.py

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8 -*-
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('api_admin', '0006_catalog'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='apiaccessrequest',
+            name='status',
```

Comment:
The DB schema isn't changed, but the model state is since the help text was changed. That being said, I'll change to using the fieldsets approach, so this gets deleted anyway.